device/celestica/x86_64-cel_seastone-r0/sonic_platform/helper.py

```diff
@@ -15,7 +15,7 @@ def __init__(self):
         (self.platform, self.hwsku) = device_info.get_platform_and_hwsku()
 
     def is_host(self):
-        return os.system(HOST_CHK_CMD) == 0
+        return subprocess.call(HOST_CHK_CMD) == 0
```

Comment:
Please consider changing the is_host function implementation as suggested above.